exporters/metrics/src/main/java/com/google/cloud/opentelemetry/metric/MetricExporter.java

```diff
@@ -96,62 +93,73 @@ private static MetricExporter createWithCredentials(
 
   @Override
   public CompletableResultCode export(Collection<MetricData> metrics) {
-    List<TimeSeries> allTimesSeries = new ArrayList<>();
-
-    for (MetricData metricData : metrics) {
-
-      // We are expecting one point per MetricData
-      if (metricData.getPoints().size() != 1) {
-        logger.error(
-            "There should be exactly one point in each metricData, found {}",
-            metricData.getPoints().size());
-        continue;
+    // General Algorithm for export:
+    // 1. Iterate over all points in the set of metrics to export
+    // 2. Attempt to register MetricDescriptors if not already registered.
+    // 3. Fire the set of time series off.
+    MetricTimeSeriesBuilder builder =  new AggregateByLabelMetricTimeSeriesBuilder(projectId);
+    for (final MetricData metricData : metrics) {
+      // Extract all the underlying points.
+      switch(metricData.getType()) {
+        case LONG_GAUGE:
+          for(LongPoint point : metricData.getLongGaugeData().getPoints()) {
```

Comment:
I'm not too familiar with the Java SDK – in practice, is there always just one point?